airflow/providers/amazon/aws/hooks/emr.py

```diff
@@ -202,57 +202,6 @@ def conn(self):
         """Get the underlying boto3 EmrServerlessAPIService client (cached)"""
         return super().conn
 
-    # This method should be replaced with boto waiters which would implement timeouts and backoff nicely.
-    def waiter(
```

Comment:
Needs to have deprecation warning before we can remove it
Also we need ro make it backward compatible by calling the function in the new location

Reply:
No one should be using this method besides the EmrServerless operators, but that's a good catch. I'll fix that. Thanks!